hack/build-operator-bundle.sh

```diff
@@ -26,3 +26,30 @@ docker build -f bundle.Dockerfile -t "${BUNDLE_FULL_IMAGE_NAME}" .
 echo
 echo "Pushing Operator bundle image to image registry..."
 docker push "${BUNDLE_FULL_IMAGE_NAME}"
+
+CSI_CR_URL="https://raw.githubusercontent.com/IBM/ibm-block-csi-operator/${BLOCK_CSI_RELEASE}/config/samples/${BLOCK_CSI_CR_FILE}"
+if curl --head --silent --fail "${CSI_CR_URL}" 2> /dev/null; then
+  echo "CSI release is GAed. Using official images"
+else
+  echo "CSI tag doesn't exist yet, cloning CSI GitHub repository"
+  oldPWD=$(pwd)
+  if [ ! -d "${DEFAULT_CSI_LOCAL_PATH}" ]
+  then
+      git clone "${DEFAULT_CSI_GIT_PATH}"
+      cd "${DEFAULT_CSI_LOCAL_PATH}"
+  else
+      cd "${DEFAULT_CSI_LOCAL_PATH}"
+      git pull
+  fi
+
+  cd "${DEFAULT_CSI_DOCKERFILE_PATH}"
+  echo "Building and pushing CSI bundle image using ${DEFAULT_CSI_DOCKERFILE_PATH}/${DEFAULT_CSI_DOCKERFILE_NAME}"
+  docker build -f "${DEFAULT_CSI_DOCKERFILE_NAME}" -t "${DEFAULT_CSI_DEVELOP_IMAGE}:${IMAGE_TAG}" .
```

Comment:
if I understand correctly, for non-GA versions we are downloading `develop` tag and building the bundle image using the dockerfile located in the relevant image version folder, right?
But the operator image in the CSV will be the GA one, which still does not exist no? do we need to edit the CSV file as we did for the CR, before building the bundle?